core/src/main/resources/lib/form/textarea/textarea.js

```diff
@@ -48,7 +48,14 @@ Behaviour.specify("DIV.textarea-preview-container", 'textarea', 100, function (e
         showPreview.onclick = function() {
             // Several TEXTAREAs may exist if CodeMirror is enabled. The first one has reference to the CodeMirror object.
             var textarea = e.parentNode.getElementsByTagName("TEXTAREA")[0];
-            var text = textarea.codemirrorObject ? textarea.codemirrorObject.getValue() : textarea.value;
+            var text = "";
+            //Textarea object will be null if the text area is disabled.
+            if (textarea == null) {
+                textarea = e.parentNode.getElementsByTagName("pre")[0];
```

Comment:
```suggestion
                textarea = e.parentNode.getElementsByClassName("jenkins-readonly")[0];
```
I can do this. Because of [l:possibleReadOnlyField](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/possibleReadOnlyField.jelly) I don't have too much to go on

Reply:
This is much better IMO and more robust. I suggest adding a check to verify that textarea is not undefined